app-steady/app-steady/FirstViewController.swift

```diff
@@ -8,27 +8,65 @@
 
 import UIKit
 
-class FirstViewController: UIViewController {
+class FirstViewController: UIViewController,UIPickerViewDataSource,UIPickerViewDelegate {
     
-    @IBOutlet weak var questionLabel: UILabel!
-    @IBOutlet weak var nextQuestionButton: UIButton!
+    @IBOutlet weak var answerPicker: UIPickerView!
+    var questionLabel = UILabel(frame: CGRectMake(50, 150, 200, 21))
+    var answerLabel = UILabel(frame: CGRectMake(50, 200, 200, 21))
+    let nextQuestionButton = UIButton(type: UIButtonType.System)
     
     var mainQuestion = nextQuestion()
     
+    let pickerData = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"]
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
-        questionLabel.text = mainQuestion.askQuestion
+        
+        questionLabel.textAlignment = NSTextAlignment.Center
+        questionLabel.text = "testing"
+        
+        answerLabel.textAlignment = NSTextAlignment.Center
+        answerLabel.text = ""
+        
+        nextQuestionButton.setTitle("Next Question", forState: UIControlState.Normal)
+        nextQuestionButton.frame = CGRectMake(50, 400, 200, 21)
+        nextQuestionButton.addTarget(self, action: #selector(self.nextQuestionButtonPressed), forControlEvents: UIControlEvents.TouchUpInside)
+        
+        self.view.addSubview(nextQuestionButton)
+        self.view.addSubview(questionLabel)
```

Comment:
See how we add the subview here. If you uncomment these lines both will disappear